test/request-response-samples/GetSubscription__should_handle_ref_minOccurs_and_maxOccurs/response.json

```diff
@@ -0,0 +1,13 @@
+{
+  "Status": {
+    "Result": "OK"
+  },
+  "Subscriptions": {
```

Comment:
Why `Subscriptions` is not an array, like it was described in [1168](https://github.com/vpulim/node-soap/pull/1168)
```json
  "Subscriptions": [{
    "Subscription": [
      {
        "SubscriptionID": "a24610ec-4069-4efc-a87e-5ab1d944ec5f",
        "TerminationTime": "2024-10-31T20:00:27.567+01:00"
      }
    ]
  }]
```

Reply:
- Making Subscriptions an array is not backward compatible. Without my change, we can still get an array of `Subscription`s if there are 2 or more subscriptions and it is returned as:

```xml
  "Subscriptions": {
    "Subscription": [
      {
        "SubscriptionID": "a24610ec-4069-4efc-a87e-5ab1d944ec5f",
        "TerminationTime": "2024-10-31T20:00:27.567+01:00"
      },
      {
        "SubscriptionID": "c24610ec-4069-4efc-a87e-5ab1d944ec5d",
        "TerminationTime": "2025-10-31T20:00:27.567+01:00"
      },
    ]
  }
```
- The XSD in #1168 is also quite different from the one here. The element with ref="tns:DummyList" suggests a list of a list of element with ref="DummyValues".
```xml
<!-- From #1168:  List of List of DummyValues-->
<xs:elements ref="tns:DummyList" minOccurs="1" maxOccurs="10" />
<xs:element name="DummyList">
  <xs:complexType> 
    <xs:sequence>
      <xs:element ref="DummyValues" minOccurs="1" maxOccurs="10" />
    </xs:sequence>
  </xs:complexType>
</xs:element>
```
The XSD in this PR suggests `Subscriptions` is a list of `Subscription`.

Reply:
okay, that makes sense.